src/ngx_wasmx.h

```diff
@@ -3,6 +3,7 @@
 
 
 #include <ngx_core.h>
+#include <ngx_wasm_shm.h>
```

Comment:
If use this header to the wasmx module, then it needs to be moved the the wasmx module sources in `config` (and become `ngx_wa_shm.h,c`) - the wasmx module can be built for IPC only today, so no `ngx_wasm` symbols would be available.

Reply:
Oh, it already is there... My bad! So this is part of the rename leftover, no worries.